CONTRIBUTING.md

```diff
@@ -8,22 +8,33 @@ See [Superchain Upgrades] OP-Stack specifications.
 
 ## Adding a chain
 
-### Adding a standard chain
-
-#### 1. Set env vars
+### 1. Set env vars
 
 To contribute a standard OP-Stack chain configuration, the following data is required: contracts deployment, rollup config, L2 genesis. We provide a tool to scrape this information from your local monorepo folder.
```

Comment:
I know you didn't make these changes but I'm not sure that the term `monorepo` is heavily used to refer to the `optimism` repo outside of OPLabs; for clarity it might help explicitly saying which repo we're talking about.

Reply:
[b29f2e6](https://github.com/ethereum-optimism/superchain-registry/pull/140/commits/b29f2e6aad66df96bf82180802a4d8b75e62783f)